tests/fixtures/observability/monitoring.py

```
@@ -0,0 +1,89 @@
+
+@pytest.fixture()
+def audit_logs(session_start_time):
+    """
+    Get audit logs names filtered by session start time.
+
+    Only returns audit logs that are relevant to the current test session:
+    - The active audit.log file
+    - Rotated files with timestamps >= session_start_time
+    - The immediately previous rotated file (to catch events just before session start)
+    """
+    output = subprocess.getoutput(
```

Ruff (0.15.21)

[error] Starting a process with a shell, possible injection detected

(S605)